packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -25,24 +30,100 @@ const notFinished: AWSCDKAsyncCustomResource.IsCompleteResponse = {
   IsComplete: false,
 };
 
+// #region Entry points
+export const onEvent = cfnResponse.safeHandler(onEventHandler);
+export const isComplete = cfnResponse.safeHandler(isCompleteHandler);
 /**
- * Util function to log especially for nested objects and arrays
- * @param msg
- * @param other arrays of arguments to be logged
+ * Handler for requests sent from CloudFormation for custom resource.
+ * This is the entry point of the custom resource.
+ *
+ * Note: This is adapted from the AWS CDK's provider framework.
+ * https://github.com/aws/aws-cdk/blob/11621e7/packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/framework.ts
+ * @param cfnRequest Request received from CloudFormation
  */
-const log = (msg: string, ...other: any[]) => {
-  console.log(
-    msg,
-    other.map((o) => (typeof o === 'object' ? JSON.stringify(o, undefined, 2) : o)),
-  );
-};
+// eslint-disable-next-line func-style
+async function onEventHandler(cfnRequest: AWSLambda.CloudFormationCustomResourceEvent): Promise<void> {
+  const sanitizedRequest = { ...cfnRequest, ResponseURL: '...' } as const;
```

Comment:
nit for future pr to follow existing redaction style

```suggestion
  const sanitizedRequest = { ...cfnRequest, ResponseURL: '[redacted]' } as const;
```

Reply:
Addressed in 329894fdf8209141716f15c4c56d77944d873239